### ISSUE TITLE
Unformated output

### ISSUE THREAD

**Issue by joseildofilho** [thosakwe/vim-flutter#27]
Hi, i'm getting this weird formated output, is it normal ?
![image](https://user-images.githubusercontent.com/8854932/84595591-ffb4f780-ae2e-11ea-93fb-4672987ec907.png)

**Comment by thosakwe** [comment]
Can you leave some instructions on how to reproduce this? What OS, what device, what version of Vim, etc.?

**Comment by joseildofilho** [comment]
@thosakwe 
I'm using Nvim 0.4.3
Arch Linux
to get this screen above, i execute `:FlutterRun`, then `FlutterTab`

**Comment by x41lakazam** [comment]
Mine is missing some newlines.
```
Launching lib/main.dart on GM1913 in debug mode...
Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          
Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          

Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          lib/home.dart:2:8: Error: Expected ';' after this.
import 'BootcampsList.dart'
       ^^^^^^^^^^^^^^^^^^^^
Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          lib/home.dart:2:8: Error: Expected ';' after this.
Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          lib/home.dart:2:8: Error: Expected ';' after this.

Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          lib/home.dart:2:8: Error: Expected ';' after this.lib/BootcampsList.dart:3:44: Error: Can't find '}' to match '{'.
class BootcampsList extends StatelessWidget{
                                           ^
Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          lib/home.dart:2:8: Error: Expected ';' after this.lib/BootcampsList.dart:3:44: Error: Can't find '}' to match '{'.
Launching lib/main.dart on GM1913 in debug mode...Running Gradle task 'assembleDebug'...                          lib/home.dart:2:8: Error: Expected ';' after this.lib/BootcampsList.dart:3:44: Error: Can't find '}' to match '{'.
```

**Comment by mracos** [comment]
This was fixed for me after #21
